vllm/v1/executor/multiproc_executor.py

```diff
@@ -268,20 +332,20 @@ def collective_rpc(  # type: ignore[override]
             send_method = cloudpickle.dumps(method, protocol=pickle.HIGHEST_PROTOCOL)
         self.rpc_broadcast_mq.enqueue((send_method, args, kwargs, output_rank))
 
-        workers = (
-            (self.workers[output_rank],) if output_rank is not None else self.workers
-        )
+        response_mqs = self.response_mqs
+        if unique_reply_rank is not None:
+            response_mqs = [self.response_mqs[unique_reply_rank]]
```

Comment:
Use `output_rank` here, also cheaper to use singleton tuple.
```suggestion
        response_mqs: Sequence[MessageQueue] = self.response_mqs
        if output_rank is not None:
            response_mqs = (response_mqs[output_rank],)
```